component/button/button.css

```diff
@@ -0,0 +1,107 @@
+.btn {
+    display: inline-block;
+    font-size: 1rem;
+    padding:0.3125rem 1rem;
```

Comment:
for values less than 10px use absolute units

Reply:
yes will do the changes

Reply:
Thanks